extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/sql/HllSketchSqlAggregatorTest.java

```diff
@@ -0,0 +1,402 @@
+                                                       )
+                                                       .setAggregatorSpecs(
+                                                           Collections.singletonList(
+                                                               new HllSketchBuildAggregatorFactory(
+                                                                   "a0:a",
+                                                                   "cnt",
+                                                                   null,
+                                                                   null
+                                                               )
+                                                           )
+                                                       )
+                                                       .setPostAggregatorSpecs(
+                                                           ImmutableList.of(
+                                                               new FinalizingFieldAccessPostAggregator("a0", "a0:a")
+                                                           )
+                                                       )
+                                                       .setContext(QUERY_CONTEXT_DEFAULT)
+                                                       .build()
+                                       )
+                                   )
+                                   .setInterval(new MultipleIntervalSegmentSpec(ImmutableList.of(Filtration.eternity())))
+                                   .setGranularity(Granularities.ALL)
+                                   .setAggregatorSpecs(Arrays.asList(
+                                       new LongSumAggregatorFactory("_a0:sum", "a0"),
+                                       new CountAggregatorFactory("_a0:count")
+                                   ))
+                                   .setPostAggregatorSpecs(
+                                       ImmutableList.of(
+                                           new ArithmeticPostAggregator(
+                                               "_a0",
+                                               "quotient",
+                                               ImmutableList.of(
+                                                   new FieldAccessPostAggregator(null, "_a0:sum"),
+                                                   new FieldAccessPostAggregator(null, "_a0:count")
+                                               )
+                                           )
+                                       )
+                                   )
+                                   .setContext(QUERY_CONTEXT_DEFAULT)
+                                   .build();
+
+      Query actual = Iterables.getOnlyElement(queryLogHook.getRecordedQueries());
+
+      // Verify query
+      Assert.assertEquals(expected, actual);
+
+    }
+    catch (Exception e) {
```

Comment:
It's same here and in `ThetaSketchSqlAggregatorTest`, `DoubleSketchSqlAggregatorTest`, and `FixedBucketsHistogramQuantileSqlAggregator`. Please check them.

Reply:
Fixed, thanks, I removed it from those files as well as `QuantileSqlAggregatorTest`.